Sources/Identifiable.swift

```diff
@@ -16,9 +16,10 @@
     extension UITableViewCell: Identifiable {
     }
 
-    extension UICollectionViewCell: Identifiable {
+    extension UITableViewHeaderFooterView: Identifiable {
     }
 
-    extension UITableViewHeaderFooterView: Identifiable {
+    extension UICollectionReusableView: Identifiable {
```

Comment:
This is enough to add support for UICollectionViewCell as well, that's why it was removed.